src/core/api/option_parsers.ts

```diff
@@ -547,6 +550,7 @@ function parseLoadVideoOptions(
   const manifestUpdateUrl = options.transportOptions?.manifestUpdateUrl;
   const minimumManifestUpdateInterval =
     options.transportOptions?.minimumManifestUpdateInterval ?? 0;
+  const audioTrackSwitchingMode = options.transportOptions?.audioTrackSwitchingMode ?? "smooth";
```

Comment:
We usually put default values in `src/config.ts`.

Look at `DEFAULT_MANUAL_BITRATE_SWITCHING_MODE` as an example

Reply:
Thanks, I see now, I have done it in the new try